website/tests/e2e.fixture.ts

```diff
@@ -91,8 +93,36 @@ export const testSequenceCount: number =
 
 const testUserTokens: Record<string, TokenCookie> = {};
 
+async function ensureKeycloakIsReachable(issuerUrl: string) {
+    try {
+        const response = await axios.get(issuerUrl);
+        if (response.status === 200) {
+            e2eLogger.debug(`Server reachable at ${issuerUrl}.`);
```

Comment:
Should we have more specific error messages for all the errors here, i.e. mention keycloak? It's not obvious from the message afaict?

Reply:
mentioned Keycloak now: https://github.com/loculus-project/loculus/pull/2995/commits/2e17caacd9d52d7427521ab1f914624a23c857db